tests/cross/src/test/scala/tests/pc/PcRenameSuite.scala

```diff
@@ -488,4 +488,12 @@ class PcRenameSuite extends BasePcRenameSuite {
        |  } yield b
        |""".stripMargin,
   )
+
+  check(
```

Comment:
Would it be a good idea to add a test where the end marker has incorrect (not matching) name and shouldn't be renamed? :slightly_smiling_face:

Reply:
why not :)

Reply:
Thank you! :+1: